cypress/integration/basic/simulation-rules/continent-ocean-collision.js

```diff
@@ -0,0 +1,95 @@
+import Model from "../../../../js/plates-model/model";
+import presets from "../../../../js/presets";
+import * as modelHelper from "../../../../js/plates-model/model-test-helpers";
+import { getImageData } from "../../../../js/utils";
+
+let modelImgData = null;
+
+before(done => {
+  getImageData("/" + presets.continentOceanCollision.img, (imgData) => {
+    modelImgData = imgData;
+    done();
+  });
+});
+
+const delta = 0.01;
+let model = undefined;
+let rockLayers = undefined;
+
+describe("Continent Ocean Collision model", () => {
+
+  it("checks initial loading of the continent-ocean collision model", () => {
+
+    model = new Model(modelImgData, presets.continentOceanCollision.init);
+
+    // Left plate - continent, ocean
+    expect(modelHelper.getFieldElevation(model, 0, 2773)).to.be.closeTo(0.55, delta);
+    expect(modelHelper.getFieldElevation(model, 0, 2458)).to.be.closeTo(0.03, delta);
+
+    // Right plate - ocean
+    expect(modelHelper.getFieldElevation(model, 1, 967)).to.be.closeTo(0.03, delta);
+  });
+
+  it("checks rock layers of left plate at the convergent boundary", () => {
```

Comment:
This might be personal/opinionated, but I tend to think that `it` refers to the tested subject. In this case "Continent Ocean Collision model". So, I'd probably say something like:
`it("sets rock layers of left plate at the convergent boundary correctly", () => {`
In your case, `it` seems to refer to the test itself. But if you prefer to organize tests that way, I don't think it matters to me. :wink: I can also see that some of the tests in this repo use a similar convention already.